### PR TITLE
Feature/1436 rename variable

### PR DESCRIPTION
**Pull request by clarkhywh**
**What's in the PR**
*Distinguish between local and global variables

**How to test manually**
* ...

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java [hunk 1]
```diff
@@ -133,6 +133,7 @@
     private static final Logger LOG = LoggerFactory.getLogger(ActiveLearningSidebar.class);
 
     // Wicket component IDs used in the HTML file
+    //ssssssss
```
Comment (munterkalmsteiner) [inline inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java]: Please delete

File: inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js [hunk 2]
```diff
@@ -13,13 +12,14 @@
 
 //
 
+
 // Generated by CoffeeScript 1.6.3
```
Comment (munterkalmsteiner) [inline inception-html-editor/src/main/java/de/tudarmstadt/ukp/inception/htmleditor/annotatorjs/resources/annotator-full.js]: This is an automatically generated file and should therefore not be changed manually.

**Comment by munterkalmsteiner** [comment]
Please request a review from me when you are done with the requested changes.

**Comment by clarkhywh** [comment]
sorry, I changed the master branch and I don't know how to solve this. So I forked the project again and rebuild the pull request

**Pull request closed without merging**